### DISCUSSION TITLE
Tooltip with Icon - ref error

### DISCUSSION THREAD

**Discussion by spidertwin2** [chakra-ui/chakra-ui#3578] [Q&A] [answered]
hi!

I am unable to add a simple Tooltip to Icon.  Error - Function components cannot be given refs.

```
<Tooltip label="like" hasArrow placement="top">
    <Icon as={BiLike} />
</Tooltip>
```

[Sandbox](https://codesandbox.io/s/competent-banach-s4z0j?file=/src/index.js)

Thanks :leaves:

**Comment by malashevskyi** [comment, marked as answer]
Hi :wave:
From docs:
![](https://i.imgur.com/ESBihBb.png)

```javascript
<Tooltip label="like" hasArrow placement="top">
  <span><Icon as={BiLike} /></span>
</Tooltip>
```

**Reply by spidertwin2** [reply]
Спасибо большое :evergreen_tree:
